### PR TITLE
PR-4-Proy-4 - Rails Blog App - Controllers

### PR DESCRIPTION
**Pull request by ivanmvh**
# Rails Blog App - Controllers
In this project, we will create all controllers and views for our Blog app. As we won't be processing any data yet, we will add simple placeholders for our views with plain HTML.

- ### Learning objectives
  - Use controllers to handle requests and render empty views.
  - Understand Rails naming conventions.
  - Use params from browser requests in a safe way.

### Project requirements
-  **Important:** we do NOT use `rails generate scaffold` command. The point of this project is to show we how to set up controllers by ourself.
-  Take a look at the wireframes below. Make sure that our app handles the URLs used in the wireframes:  

image

-  For each URL we should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Make sure that we added code to handle **only** the 4 required URLs.

###### Make sure that when we run the `rails routes` command in our console, we can see the following output:

 |    Prefix       |   Verb   |     URI   Pattern                                  |   Controller#Action    |    Description           | 
 | -------------| --------| ----------------------------------------|  ----------------------  |  ---------------------| 
 | user_posts  |   GET    |  /users/:user_id/posts(.:format)         |     posts#index            |    List of user posts | 
 | user_post    |  GET    |   /users/:user_id/posts/:id(.:format)   |     posts#show            |   Show a user post  | 
 | users          |   GET    |  /users(.:format)                                 |         users#index       |        List of users      | 
 |  user           |  GET    |  /users/:id(.:format)                            |         users#show        |       Show a user      | 
 | root            |   GET    |  /                                                       |         users#index        |  root-List of users   | 

### General requirements (Don't forget)
- Make sure that there are no linter errors.
- Make sure that we used correct flow Gitflow.
- Make sure that we documented our work in a professional way.
- Follow our list of best practices for Ruby.

@ivanmvh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@
 
 **Blog App** The Blog app is classic example of a blog website. A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
```
Comment (arslanbisharat) [inline README.md]: - [x] Kindly add .vscode to your gitignore as it makes project unclean.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @ivanmvh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ivanmvh) [reply README.md]: It's ready. Thank you.

**Review by omarsalem7 — APPROVED** [review]
Hi @ivanmvh 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**